ament_clang_format/cmake/ament_clang_format_lint_hook.cmake

```diff
@@ -0,0 +1,14 @@
+file(GLOB_RECURSE _source_files FOLLOW_SYMLINKS
+  "*.c"
+  "*.cc"
+  "*.cpp"
+  "*.cxx"
+  "*.h"
+  "*.hh"
+  "*.hpp"
+  "*.hxx"
+)
+if(_source_files)
+  message(" - Added test 'clang_format' to check C / C++ code style")
+  ament_clang_format(clang_format)
```

Comment:
Since all registered test targets are being used by `ctest` afterwards it is kind of weird / redundant if they all have `test` in it. Therefore I would rather not add it as a prefix / suffix.